radar/platforms-and-services/azure-application-gateway.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Azure Application Gateway"
+ring: adopt
+quadrant: "platforms-and-aoe-services"
+tags: [cloud, azure, networking]
+---
+
+[Azure Application Gateway](https://learn.microsoft.com/en-us/azure/application-gateway/overview) is a regional web traffic load balancer (OSI layer 7) service. It facilitates the optimization of performance, reliability, and security for web applications. Equipped with features such as SSL termination, URL-based routing, session affinity and Web Application Firewall (WAF) integration, Application Gateway ensures efficient traffic distribution, enhanced application scalability, and protection against common web vulnerabilities. It also can be used as ingress controller for [AKS](https://azure.microsoft.com/en-us/products/kubernetes-service).
+
+### Use cases
+
+Opt for Application Gateway when you need to load balance web applications (HTTP/HTTPS) that can be exposed on internet or not. Remember it is a regional service, so if your application runs in multiple regions you need to replicate the Application Gateway as well (opposed to the [Front Door](https://learn.microsoft.com/en-us/azure/frontdoor/front-door-overview)).
+Application Gateway is often put in front of an [APIM](https://azure.microsoft.com/en-us/products/api-management) instance for its WAF capabilities, DDoS protection and SSL offloading.
```

Comment:
It's not clear if we should put it in front of the APIM or if it's a weak suggestion.

Reply:
It's a best practice and I added some example: `for its WAF capabilities, DDoS protection and SSL offloading.`
Application Gateway is a load balancer, APIM is not